web/docs/auth/auth-hooks.md

```diff
@@ -287,6 +296,86 @@ export const onAfterSignup: OnAfterSignupHook = async ({
 
 Read more about the data the `onAfterSignup` hook receives in the [API Reference](#the-onaftersignup-hook).
 
+### Executing code after a user verifies the email
+
+Wasp calls the `onAfterEmailVerified` hook once a user's email has been successfully verified.
+
+The `onAfterEmailVerified` hook is useful for triggering actions in response to the verification event — such as sending a welcome email or syncing user data with a third-party service.
+
+The `onAfterEmailVerified` hook receives a `ProviderId` object, which always contains the `"email"` provider along with the user ID (i.e., the verified email address). Combined with the `user` object, this makes it easy to perform personalized actions upon email verification.
+
+Works with <EmailPill />
+
+<Tabs groupId="js-ts">
+<TabItem value="js" label="JavaScript">
+
+```wasp title="main.wasp"
+app myApp {
+  ...
+  auth: {
+    ...
+    onAfterEmailVerified: import { onAfterEmailVerified } from "@src/auth/hooks",
+  },
+}
+```
+
+```js title="src/auth/hooks.js"
+import { emailSender } from 'wasp/server/email'
+
+export const onAfterEmailVerified = async ({ providerId }) => {
+  const info = await emailSender.send({
+    from: {
+      name: 'John Doe',
+      email: 'john@doe.com',
+    },
+    to: provider.providerUserId,
+    subject: 'Thank you for verifying your email!',
+    text: 'Your email has been successfully verified!',
+    html: 'Your email has been successfully verified!',
+  })
+  // Store that email was sent successfully, otherwise schedule it for later
```

Comment:
I'd drop this, it's a business logic detail unrelated with the hook. I'd just do `// ...`